releasenotes/notes/add-openai-responses-chatgenerator-52ca7457a4e61db1.yaml

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This needs to be a highlight IMHO. It's no easy feat to have Responses API integrated well into Haystack ecosystem.   I would also add more examples. Perhaps how to use their web_search and our tool for weather that always returns sunny and 22C :-)